File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_3/The solution/C/Q_2.c
#include <stdio.h>

void main(void)
{
    unsigned short age; // مافيش عر بالسالب او اكبر من 65,535
    char name[10];

    // Read age from user
    printf("Please enter your age: ");
    scanf("%d", &age);
    // Read name from user
    printf("Please enter your first name: ");
    scanf("%s", name); // ما بعتش العنوان لان المتغير نيم من نع بوينتر فا هوا عنوان اصلا

    // Check age and print result
    (age >= 18)? printf("\nWelcome %s", name):printf("\nSorry, this service is not available for your age group\nGoodbye ^-^\n");
}
// Programming is very, very fun 🖤

File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_3/The solution/Java/Q_2.java
package com.company; // If you encounter a problem running the code, delete this line

import java.util.Scanner;

public class Q_2 { // If you encounter a problem running the code, change the file name to "Q_1.java"
    public static void main(String[] args){
        short age;
        String name;
        Scanner input = new Scanner(System.in);

        // Read age from user
        System.out.print("Please enter your age: ");
        age = input.nextShort();
        // Read name from user
        System.out.print("Please enter your first name: ");
        name = input.next(); // read one token as a string

        // Check age and print result
        System.out.printf("%s", (age >= 18)? ("\nWelcome "+name) : "\nSorry, this service is not available for your age group\nGoodbye ^-^");

    }
}
// Programming is very, very fun 🖤